src/modules/lighthouse/auditUrl.ts

```diff
@@ -0,0 +1,38 @@
+import ora from 'ora'
+import chalk from 'chalk'
+
+import log from '../../logger'
+import { lighthouse, workspaces } from '../../clients'
+import { getAccount, getWorkspace } from '../../conf'
+import { TableGenerator } from './TableGenerator'
+
+const [account, currentWorkspace] = [getAccount(), getWorkspace()]
+
+async function isProdutionWorkspace(): Promise<boolean> {
+  const meta = await workspaces.get(account, currentWorkspace)
+  return meta.production
+}
+
+export default async (url: string, option: any) => {
+  if (await isProdutionWorkspace()) {
+    log.error(`You can not run lighthoust audits on production workspaces.`)
+    return
+  }
+
+  const spinner = ora(`Running Lighthouse on url: ${chalk.blue(url)}`).start()
+  try {
+    const report = await lighthouse.runAudit(url)
+    spinner.stop()
+
+    if (option.j || option.json) {
+      console.log(JSON.stringify(report, null, 1))
+    } else {
+      const table = new TableGenerator()
+      table.addReportScores(report)
+      table.show()
+    }
+  } catch (error) {
+    spinner.stop()
+    log.error(error)
```

Comment:
For telemetry purposes:
```ts
TelemetryReporter.createAndRegisterErrorReport({ originalError: error })
```

Reply:
Awesome! Should I add the TelemetryReporter only when there was an error? Or also in success command executions?

How should I send these error to TelemetryReporter, because `createAndRegisterErrorReport` does not exists in TelemetryReport :/ Is there any doc for telemetry? :)

Reply:
Just to let it registered here
``` js
TelemetryCollector.createAndRegisterErrorReport({ originalError: error })
```